src/test/resources/alma/(DE-605)HT015671602.json

```diff
@@ -155,8 +155,8 @@
       "label" : "Dewey-Dezimalklassifikation"
     }
   }, {
-    "type" : [ "Concept" ],
     "label" : "FRBR (Conceptual model)",
+    "type" : [ "Concept" ],
     "source" : {
```

Comment:
As argued before, don't type something as skos concept when it's lacking at least the most crucial id.

Reply:
You argued another case. Here: If a subject is a Library of Congress Subject Heading as in this case it is a controlled vocab and counts as a `concept`. It would be nice though to add the ID but we do not have a mapping yet https://id.loc.gov/search/?q=memberOf:http://id.loc.gov/authorities/subjects/collection_LCSHAuthorizedHeadings

Reply:
https://id.loc.gov/authorities/subjects/sh2007002541.html